packages/fxa-content-server/app/scripts/templates/account_recovery_confirm_key.mustache

```diff
@@ -51,6 +51,6 @@
     {{/isLinkExpired}}
 </div>
 
-<!-- New L10N Strings
-{{#t}}Please enter the one time use account recovery key you stored in a safe place to regain access to your Mozilla account.{{/t}}
+<!-- Old L10N Strings - TBR
```

Comment:
@bcolsson Keeping the old strings in comments will not work to keep them available in source? I'm not sure what our alternative might be for these content-server files...

Reply:
This string doesn't show up in my diff, so this is fine/preserved.

Reply:
I should note that the extraction script extracts strings from the .js files. A completely non-thorough spot check seems to indicate this is most of the strings triggering my check.

Reply:
I checked on Pontoon that the strings we'd initially added in comments were indeed translated (that was fine), but now I'm having doubts if these strings we pulled into comments will be mapped properly if they're needed as backup... @dschom

Reply:
@bcolsson Is possible the diff is just confusing? For example, if you look at this you can see the original string is still there and has been added. We are just swapping strings. I'll try to come through and see if there any that are actually missing, but I think most have simply been swapped between the commented out code that we introduced in the prior PR. For example, here you can see that `We detected suspicious behavior on your Firefox account. To protect your`. was moved from the top of the file to the bottom of the file:

![image](https://github.com/mozilla/fxa/assets/94418270/78838027-971a-4ca8-a0cd-fb0639f20dee)
****

Reply:
@bcolsson I found three strings in js files that were not accounted for. I kept the [commit](https://github.com/mozilla/fxa/pull/15908/commits/1f09ae9d4993f1449c356bd82f9eaec1ef5d3f92) separate for now so they are easy to spot. I think all legacy strings are now present in the extracted output.

Reply:
@bcolsson I was thinking about this, and just wanted to clarify something. Does the order of the strings matter? If so, I can probably just take a slightly different approach to how the commented out strings are added. So instead of doing something like this
```
<div>
   <h1>{{t}}Mozilla account{{/t}}</h1>
</div>
<!-- Old strings
{{t}}Firefox Accounts{{/t}}
-->
```

To preserve file order, could we do this?
```
<div>
   <h1>
      {{t}}Mozilla account{{/t}}<!-- {{t}}Firefox Accounts{{/t} -->
  </h1>
</div>
```

This way the only thing that changes is where the comment directives are. And the line numbers wouldn't stay the same.